test/test_compare256_rle.cc

```diff
@@ -52,12 +52,12 @@ static inline void compare256_rle_match_check(compare256_rle_func compare256_rle
 
 TEST_COMPARE256_RLE(c, compare256_rle_c, 1)
 
-#ifdef UNALIGNED_OK
+#if BYTE_ORDER == LITTLE_ENDIAN && OPTIMAL_CMP >= 32
```

Comment:
I don't think you need to care about `OPTIMAL_CMP` for unit tests. These should just all be run.

Reply:
Wouldn't that break the test, since the function it wants to call is not compiled?

What I added here was the missing LITTLE_ENDIAN check, since I noticed we don't compile these functions without that.

Reply:
I see, ok.